src/service/data_products/common_models.py

```diff
@@ -27,6 +27,12 @@ class DBCollection(BaseModel):
     view associated with it.
     """
 
+    generic_view_required: bool = False
+    """
+    Whether the collection requires a generic ArangoSearch view. If so, the view will be automatically 
+    created by the service manager. `view_required` flag is ignored if this flag is set.
```

Comment:
```suggestion
    Whether the collection requires a generic ArangoSearch view. `view_required` flag is ignored if this flag is set.
```
The service code really shouldn't know the service manager exists